Todo-script.js

```diff
@@ -1,47 +1,52 @@
-// Dodavanje "close" dugmeta za svaku stavku liste
-const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
-var variable = $("#userInput");
-
-// Ciljanje <ul> preko klase "todo-list"
-var todoList = $(".todo-list");
-
-$("li").each(function () {
-  $(this).append(SPAN_CLOSE);
-});
-
-// Klik na "close" dugme skriva trenutnu stavku
-todoList.on("click", ".close", function () {
-  $(this).parent().css("display", "none");
-});
-
-// Klik na stavku označava je kao "checked"
-todoList.on("click", "li", function () {
-  $(this).toggleClass("checked");
-});
-
-// Dodavanje nove stavke na listu
-$(".add-btn").click(function () {
-  var inputValue = variable.val();
-  if (!inputValue) {
-    showError("You must enter something on To-do list.");
-  } else {
-    var listItem = $("<li>").text(inputValue).append(SPAN_CLOSE);
-    todoList.append(listItem);
-    variable.val("");
+window.onload = function () {
```

Comment:
I madea  mistake here, i called it on load, but when i googled it its actualy document ready:
`$( document ).ready(function() {
    console.log( "ready!" );
});`

This is a jquery function which will automatically execute , but only once the html has loaded all the data (css, scripts...)
You need to use this